src/utils/createPageNumbers.js

```diff
@@ -0,0 +1,12 @@
+import { POSTS_PER_PAGE } from "../constants/pagination";
+
+export const createPageNumbers = (totalPosts) => {
+  const pageNumbers = [];
+
+  if (typeof totalPosts === "number") {
+    for (let i = 1; i <= Math.ceil(totalPosts / POSTS_PER_PAGE); i++) {
```

Comment:
Move the `Match.ceil(totalPosts / POSTS_PER_PAGE)` to a variable like `pageNumbers` for making code cleaner.